daemon/networkdriver/portallocator/portallocator.go

```diff
@@ -3,8 +3,23 @@ package portallocator
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"net"
+	"regexp"
+	"strconv"
 	"sync"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	defaultPortRangeStart = 49153
+	defaultPortRangeEnd   = 65535
+)
+
+var (
+	BeginPortRange = 0
+	EndPortRange   = 0
```

Comment:
Yeah, agreed. I just wanted to preserve those exported variables so I didn't have to touch other modules using them. Currently they are used just in test modules though. I'll make it other way around and provide a getter for `(Begin|End)PortRange`.